net/samba4/Portfile

```diff
@@ -1,36 +1,53 @@
 PortSystem 1.0
 
 name			samba4
-version			4.0.0tp5
-revision		5
+version			4.13.0
 categories		net
 platforms		darwin
 maintainers		nomaintainer
 license			GPL-3
-description		SMB/CIFS server and client, ALPHA release
+description		SMB/CIFS server and client
 long_description	Samba is an software suite that provides seamless file \
-			and print services to SMB/CIFS clients. \
-			This is an ALPHA release!
+			and print services to SMB/CIFS clients.
 
 homepage		http://www.samba.org/
-master_sites		http://download.samba.org/samba/ftp/samba4/ \
-			ftp://de.samba.org/samba.org/samba4/
+master_sites		https://download.samba.org/pub/samba/stable
 distname		samba-${version}
-checksums           md5     dbc2e04455d67b01d2020808065b96df \
-                    sha1    8897cc7a824a594b3543245cdbbceaea32e9e08d \
-                    rmd160  1870f67aa851751e381a3b03dadfa1d0afa28a12
+checksums       rmd160  0e0efa2be30af3df3cce92b00496b296cc4c2a0d \
+                sha256  f11d52aee8db0aa50ac614143f2ccefead29e8f8ff585358792221557cade7f6
```

Comment:
We are now adding the `size` parameter to `checksums`.

Reply:
would that be in bytes? or kilobytes? the tar.gz for samba4 has a size of 18406380 bytes (18M).

Reply:
Yes, bytes. What I do to get it is to set it to 1 and then run `port -v checksum` on the Portfile locally to get the value.